airflow/providers/google/cloud/hooks/cloud_build.py

```diff
@@ -185,54 +185,9 @@ def create_build_without_waiting_for_result(
             metadata=metadata,
         )
         id_ = self._get_build_id_from_operation(operation)
-        return operation, id_
-
-    @GoogleBaseHook.fallback_to_default_project_id
-    def create_build(
```

Comment:
maybe need to deprecated it first and we should remove it only in major release

Reply:
@pankajastro 
Thank you for your review. I have reverted the changes based on your feedback.
If you happen to know, could you please tell me what is the common way to deprecate a method in Airflow?

Reply:
It depends - look at similar cases where DeprecationWarning is raised - just search in our codebase.